src/org.ts

```diff
@@ -233,7 +233,7 @@ export class Org extends AsyncCreatable<Org.Options> {
     const conn = this.getConnection();
     let isDevHub = false;
     try {
-      await conn.query('SELECT Id FROM ScratchOrgInfo');
+      await conn.query('SELECT Id FROM ScratchOrgInfo limit 1');
```

Comment:
What's cheaper a soql query or sobject describe?

Reply:
couldn't you use `singleRecordQuery`? If the query returning 2+ results is a bug we should have a test to verify we don't break this in the future

Reply:
query

Reply:
we don't care how many come back.  0 records is fine for a virgin devhub...
we only care that the obj exists at all and doesn't throw an error

Reply:
I'm setting it to `1` so busy hubs don't have to return 50000 or whatever.